apps/web/features/customization/components/widget-preview.tsx

```
@@ -28,96 +27,82 @@ export const WidgetPreview = ({ data }: WidgetPreviewProps) => {
-											</button>
-										),
-								)}
-							</div>
+						<div className="flex flex-col gap-2 overflow-x-auto p-2 px-4 pb-0 scrollbar-hide">
+							{Object.values(data.defaultSuggestions || {}).map(
+								(suggestion, idx) =>
+									suggestion && (
+										<button
+											key={idx}
+											className="whitespace-nowrap rounded-full border bg-background px-3 py-1 text-xs text-muted-foreground hover:bg-muted"
+										>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)